src/Core/Framework/Routing/Facade/RequestFacade.php

```diff
@@ -128,7 +128,7 @@ public function request(): array
      */
     public function headers(): array
```

Comment:
```suggestion
    public function headers(): ?array
    {
        if (!$this-request) {
            return null;
        }
        
        $headers = array_change_key_case($this->request->headers->all());
```

Reply:
Hi David, I changed that in my last commit. Thank you.